conn.go

```diff
@@ -79,6 +100,25 @@ func (c *Conn) Write(b []byte) (err error) {
 	return nil
 }
 
+func (c *Conn) FlushWriter() (err error) {
+	bufWriter, ok := c.writer.(*bufio.Writer)
+	if ok {
+		c.wmtx.Lock()
+		defer c.wmtx.Unlock()
+		return flushWriter(bufWriter)
+	}
+
+	return errors.New("not buffered")
```

Comment:
Create an `ErrNotBuffer` or something globally and return it Users can compare the error against it if they want.